tutorials/position-velocity-diagrams/PVDiagramPlotting.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>Optional:  add an activity. E.g., show a different path, use a different file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
decided not to do this for now